app/app.go

```diff
@@ -227,6 +227,12 @@ func Run(ctx context.Context, conf Config) (err error) {
 
 	// seenPubkeys channel to send seen public keys from validatorapi to monitoringapi.
 	seenPubkeys := make(chan core.PubKey)
+	seenPubkeysFunc := func(pk core.PubKey) {
```

Comment:
had to refactor this due to failing tests: https://github.com/ObolNetwork/charon/actions/runs/3712691482/jobs/6294500914